### ISSUE TITLE
TSP needs more usage info

### ISSUE THREAD

**Issue by JoelPasvolsky** [dwavesystems/dwave_networkx#82]
**Description**
Most TSP problems I try fail to produce a minimal route. The parameters and working range of splvable problems should be outlined. 

**To Reproduce**
The following alternative weights to the 5-node problem, EdgeView([(0, 1), (0, 2), (0, 3), (1, 2), (1, 3), (2, 3)]), all fail:

* [{'weight': 22.36},  {'weight': 10.0},  {'weight': 14.14},  {'weight': 20.0},  {'weight': 22.36}, {'weight': 22.36}]
  produces route: [3, 1, 0] (non-Hamiltonian route)
* [{'weight': 2.24}, {'weight': 1.0}, {'weight': 1.41}, {'weight': 2.0}, {'weight': 2.24}, {'weight': 2.24}]
  produces route: [1, 0, 3, 2] (a better answer is [1 2 0 3])
* [{'weight': 2}, {'weight': 4}, {'weight': 1}, {'weight': 2}, {'weight': 4}, {'weight': 3}]
  produces route: [3, 2, 1, 0] (a better answer is [2 1 0 3])

**Expected behavior**
More info on range of problem inputs, how to set lagrange, etc

**Environment:**
 - OS: Windows
 - Python version: 2.7

**Additional context**
@vgoliber

**Comment by vgoliber** [comment]
Choosing a value for a Lagrange parameter is a topic that is relevant for all of our algorithm tools (not just TSP).  I think it would be more appropriate to provide higher level guidance and/or references on this subject as a whole rather than guidance specific to each algorithm.

**Comment by JoelPasvolsky** [comment]
Agree, @vgoliber, you mentioned having references and some guidance. Please add that here for that general update. Thanks!
